pkg/tsdb/azuremonitor/azuremonitor-datasource.go

```diff
@@ -248,6 +248,7 @@ func (e *AzureMonitorDatasource) createRequest(ctx context.Context, dsInfo *mode
 
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("User-Agent", fmt.Sprintf("Grafana/%s", setting.BuildVersion))
+	req.Header.Set("Prefer", fmt.Sprintf("wait=%d", setting.DataProxyTimeout))
```

Comment:
Are you sure that this applies to the azure monitor metrics service / api? I do see documented for the analytics and insights. So perhaps it should be removed here?

Also this should perhaps also be added to insights-analytics-datasource.go ?

Reply:
I can only find it documented for analytics and insights - the metrics API would be less prone to variations in query times, so yes will remove from here + look at insights-analytics-datasource